### ISSUE TITLE
Merge with adobe-script-starter

### ISSUE THREAD

**Issue by timhaywood** [fartinmartin/adobe-lib-starter#1]
Hey this is super cool!

Like the idea of also bundling a file to be published on npm.

If you're interested, should we take a look at how we can merge this and adobe-script-starter? Wonder if I can get the same functionality working with SWC... 

Also if you want a reference on creating the npx package to setup projects, can check out:
https://github.com/motiondeveloper/create-expression-lib

**Comment by fartinmartin** [comment]
Hey @timhaywood, thanks for checking this out! Merging with `adobe-script-starter` makes total sense. I was originally messing with a fork, but was struggling with SWC—specifically in removing `export` declarations in the bundled output.

I'm also not super confident with SWC's bundling options, and think I've gone cold on SWC generally (preferring the heavily documented Rollup 😬), would love to learn from you though!

Thanks for the `npx` reference, I'll definitely be stealing more of your ideas, haha! I love the idea of taking care of house keeping (renaming, adding correct types to config, etc) with CLI prompts.

**Comment by timhaywood** [comment]
Agree rollup is probably the way to go, chose SWC mostly out of interest/experimentation 😬 Fun to use, but yeah rollup is much better documented, and easier to write plugins for.

Have you seen https://github.com/Klustre/extender/ ? Seems to be a great solution, so might archive my starter in favour of that one.

**Comment by fartinmartin** [comment]
Yeah, extender looks awesome—lots of great features! I haven't totally figured out how to use TypeScript with it yet, but haven't had too much time to play around with it.
